src/bosh-director/lib/bosh/director/addon/addon_filter.rb

```diff
@@ -11,8 +11,11 @@ def initialize(applicable_jobs, applicable_deployment_names, applicable_stemcell
         @filter_type = filter_type
       end
 
-      def self.parse(addon_filter_hash, filter_type)
+      def self.parse(addon_filter_hash, filter_type, deployment_level = false)
```

Comment:
Rather than a boolean value here, could we consider a constant representing the scope? For example, `Bosh::Director::Addon::CONTEXT_RUNTIME` (or DIRECTOR?) and `...::CONTEXT_DEPLOYMENT`? Then we could also use it in the `Addon` class, too. It makes arguments a bit more explicit and we don't have to wonder what a boolean value means.